docs/design/job_architecture.md

```diff
@@ -583,55 +550,36 @@ In kernel, as retrieved from EE2.check_job
     "retry_parent": null
 }
 ```
-#### BE Output State
+#### Job output state
 As sent to browser, includes cell info and run info
-```
-{
-    outputWidgetInfo: (if not finished, None, else...) job.get_viewer_params result
-    jobState: {
-        job_id: string,
-        status: string,
-        created: epoch ms,
-        updated: epoch ms,
-        queued: optional - epoch ms,
-        finished: optional - epoch ms,
-        terminated_code: optional - int,
-        tag: string (release, beta, dev),
-        parent_job_id: optional - string or null,
-        run_id: string,
-        cell_id: string,
-        errormsg: optional - string,
-        error (optional): {
-          code: int,
-          name: string,
-          message: string (should be for the user to read),
-          error: string, (likely a stacktrace)
-        },
-        error_code: optional - int
-        created: 1583863267000,
-        batch_id: str,
-        batch_job: bool,
-        child_jobs: array,
-        retry_ids: array,
-        retry_parent: str
-    }
-}
-```
-
-When an error occurs while preparing the job state, the output states will have the formats
 ```json
 {
-  "job_id_0": {
-    "jobState": {"job_id": "job_id_0", "status": "does_not_exist"}
-  },
-  "job_id_1": {
+    "job_id": string,
+    "outputWidgetInfo": null if job not completed; otherwise job.get_viewer_params result,
     "jobState": {
-      "job_id": "job_id_1",
-      "status": "ee2_error",
-      "updated": 1234567890, // the current epoch time
-      ... // cached job data
-      }
-  },
-  ...
+        "job_id": string,
+        "status": string,
+        "batch_id": str,
+        "batch_job": bool,

```

Comment:
What's the difference between batch id and batch job?

Reply:
batch_job = boolean indicating whether or not this is a batch job; if true, the batch_id will hold the ID for it.

Reply:
`batch_id` seems redundant to `bool(batch_job)` in that case

Reply:
I think this was the EE2 structure that was deleted, so moot

Reply:
Never mind, it's here: [Batch cells have an update mechanism that triggers a STATUS request a short period after receiving the last update from the backend.](https://github.com/kbase/narrative/blob/DATAUP-729-recovered-commits/docs/design/job_architecture.md#job-output-state)

Reply:
I think this is an ethereal question